controllers/volumereplication_controller.go

```diff
@@ -317,6 +328,50 @@ func (r *VolumeReplicationReconciler) SetupWithManager(mgr ctrl.Manager, cfg *co
 		WithEventFilter(pred).Complete(r)
 }
 
+func (r *VolumeReplicationReconciler) waitForCrds() error {
+
+	logger := r.Log.WithName("checkingDependencies")
+
+	err := r.waitForVolumeReplicationResource(logger, volumeReplicationClass)
+	if err != nil {
+		logger.Error(err, "Failed to watch VolumeReplicationClass")
+		return err
+	}
+
+	err = r.waitForVolumeReplicationResource(logger, volumeReplication)
+	if err != nil {
+		logger.Error(err, "Failed to watch VolumeReplication")
+		return err
+	}
+	return nil
+}
+
+func (r *VolumeReplicationReconciler) waitForVolumeReplicationResource(logger logr.Logger, resourceName string) error {
+
+	isVolumeReplicationResourceExists := false
```

Comment:
Suggesting to change this name. Doesn't sound correct. 
Suggestions -  `isResourcePresent`    `resourceExists`  `resourcePresent`.

Reply:
Those names do not reflect what this function does.
This function waits until the resource is showing up, it doesn't tell if it exists or not.

Reply:
since the method name is `waitForVolumeReplicationResource`,  it should be ok to remove the word `VolumeReplication`  from the variable name. Hence the above suggestions.

Reply:
done